vdev/src/testing/integration.rs

```diff
@@ -134,6 +152,10 @@ impl IntegrationTest {
     }
 }
 
+// Question: Should I inject network here or should I keep it in the compose config?
+// I am leaning towards keeping it in the compose config. Since that seems to be the
+// Compose file direct mapping, but I would like guidance on this. For now I can
+// Add to network by reading the compose config.
```

Comment:
The network block has to go into the compose config, because it has to be written out to the temporary file.